src/Microsoft.Health.Fhir.Core/Features/Conformance/CapabilityStatementBuilder.cs

```diff
@@ -112,6 +117,16 @@ public ICapabilityStatementBuilder AddRestInteraction(string systemInteraction)
             return this;
         }
 
+        public ICapabilityStatementBuilder AddRestSearchParams()
+        {
+            foreach (SearchParamComponent searchParam in SearchRestParams)
+            {
+                _statement.Rest.Server().SearchParam.Add(searchParam);
```

Comment:
We could probably get away with just doing `_statement.Rest.Server().SearchParam.Add(
                new SearchParamComponent { Name = SearchParameterNames.ResourceType, Definition = SearchParameterNames.TypeUri, Type = SearchParamType.Token });` here since the values we're adding aren't likely to change dynamically.